File: unittest/scripts/auto/js_devapi/scripts/mysqlx_collectionmodify_help.js
//@ Initialization
shell.connect(__uripwd);
var schema = session.createSchema('collectionmodify_help');
var collection = schema.createCollection('sample');
var crud = collection.modify("true")
session.close();

//@ CollectionModify help
crud.help();

//@ CollectionModify help, \? [USE:CollectionModify help]
\? CollectionModify

//@ Help on arrayAppend
crud.help('arrayAppend');

//@ Help on arrayAppend, \? [USE:Help on arrayAppend]
\? CollectionModify.arrayAppend

//@ Help on arrayDelete
crud.help('arrayDelete');

//@ Help on arrayDelete, \? [USE:Help on arrayDelete]
\? CollectionModify.arrayDelete

//@ Help on arrayInsert
crud.help('arrayInsert');

//@ Help on arrayInsert, \? [USE:Help on arrayInsert]
\? CollectionModify.arrayInsert

//@ Help on bind
crud.help('bind');

//@ Help on bind, \? [USE:Help on bind]
\? CollectionModify.bind

//@ Help on execute
crud.help('execute');

//@ Help on execute, \? [USE:Help on execute]
\? CollectionModify.execute

//@ Help on help
crud.help('help');

//@ Help on help, \? [USE:Help on help]
\? CollectionModify.help

//@ Help on limit
crud.help('limit');

//@ Help on limit, \? [USE:Help on limit]
\? CollectionModify.limit

//@ Help on merge
crud.help('merge');

//@ Help on merge, \? [USE:Help on merge]
\? CollectionModify.merge

//@ Help on modify
crud.help('modify');

//@ Help on modify, \? [USE:Help on modify]
\? CollectionModify.modify

//@ Help on patch
crud.help('patch');

//@ Help on patch, \? [USE:Help on patch]
\? CollectionModify.patch

//@ Help on set
crud.help('set');

//@ Help on set, \? [USE:Help on set]
\? CollectionModify.set

//@ Help on sort
crud.help('sort');

//@ Help on sort, \? [USE:Help on sort]
\? CollectionModify.sort

//@ Help on unset
crud.help('unset');

//@ Help on unset, \? [USE:Help on unset]
\? CollectionModify.unset

//@ Finalization
shell.connect(__uripwd);
session.dropSchema('collectionmodify_help');
session.close();

File: unittest/scripts/auto/py_devapi/scripts/mysqlx_collectionmodify_help.py
shell.connect(__uripwd)
schema = session.create_schema('collectionmodify_help')
coll = schema.create_collection('sample')
collmodify = coll.modify('1=1');
session.close()

#@ collmodify
collmodify.help()

#@ global ? for CollectionModify[USE:collmodify]
\? CollectionModify

#@ global help for CollectionModify[USE:collmodify]
\help CollectionModify

#@ collfind.array_append
collmodify.help('array_append')

#@ global ? for array_append[USE:collfind.array_append]
\? CollectionModify.array_append

#@ global help for array_append[USE:collfind.array_append]
\help CollectionModify.array_append

#@ collfind.array_delete
collmodify.help('array_delete')

#@ global ? for array_delete[USE:collfind.array_delete]
\? CollectionModify.array_delete

#@ global help for array_delete[USE:collfind.array_delete]
\help CollectionModify.array_delete

#@ collfind.array_insert
collmodify.help('array_insert')

#@ global ? for array_insert[USE:collfind.array_insert]
\? CollectionModify.array_insert

#@ global help for array_insert[USE:collfind.array_insert]
\help CollectionModify.array_insert

#@ collfind.help
collmodify.help('help')

#@ global ? for help[USE:collfind.help]
\? CollectionModify.help

#@ global help for help[USE:collfind.help]
\help CollectionModify.help

#@ collfind.merge
collmodify.help('merge')

#@ global ? for merge[USE:collfind.merge]
\? CollectionModify.merge

#@ global help for merge[USE:collfind.merge]
\help CollectionModify.merge

#@ collfind.modify
collmodify.help('modify')

#@ global ? for modify[USE:collfind.modify]
\? CollectionModify.modify

#@ global help for modify[USE:collfind.modify]
\help CollectionModify.modify

#@ collfind.patch
collmodify.help('patch')

#@ global ? for patch[USE:collfind.patch]
\? CollectionModify.patch

#@ global help for patch[USE:collfind.patch]
\help CollectionModify.patch

#@ collfind.set
collmodify.help('set')

#@ global ? for set[USE:collfind.set]
\? CollectionModify.set

#@ global help for set[USE:collfind.set]
\help CollectionModify.set

#@ collfind.unset
collmodify.help('unset')

#@ global ? for unset[USE:collfind.unset]
\? CollectionModify.unset

#@ global help for unset[USE:collfind.unset]
\help CollectionModify.unset

shell.connect(__uripwd)
session.drop_schema('collectionmodify_help');
session.close()
